### ISSUE TITLE
[Feature request] Shift+Right-Click terminal for storage capacity popup

### ISSUE THREAD

**Issue by MaxWasUnavailable** [tom5454/Toms-Storage#435]
It'd be nice if we could shift+right-click on terminals to get the used storage / total storage popup.

**Comment by ffuentesm** [comment]
This is already a thing
YOu can use JADE for the tooltip, or right click with your hand on the inventory Connector
![image](https://github.com/user-attachments/assets/b8860ce1-8c78-419b-b4f0-2dee49845c54)

**Comment by MaxWasUnavailable** [comment]
Right, as mentioned in my original post, I specifically mean the terminal - not the inventory connector. Since the inventory connector is usually obscured to keep builds clean.

I'm also not referring to the item tooltip that Jade provides. As I wrote in the post, I'm referring to the free / total popup (which you can see on the bottom of your screenshot).

It'd be a neat way to check your current capacity when you're either far away from your inventory connector and do have terminals, or when your inventory connector is obscured by other blocks.
